### PR TITLE
Matrix Storage in Ceph

### PR DESCRIPTION
**Pull request by expedited**
## Changes

* Created S3 bucket in Ceph 'synapse-storage'
* Added to homeserver.yaml cfg file to include new Ceph storage bucket 
* Included the storage provider file in Docker image, and added this file to the image PYTHONPATH

## TODO's
* Figure out how to update the s3_storage_provider file as it updates within the matrix-org repo 

Addresses issue https://github.com/ocf/matrix/issues/6

### CODE REVIEW

File: kubernetes/secrets/synapse-config/homeserver.yaml.erb [hunk 1]
```diff
@@ -43,6 +43,17 @@ log_config: "/etc/matrix/synapse.log.config"
...
 media_store_path: "/data/media_store"
 
+media_storage_providers:
+- module: s3_storage_provider.S3StorageProviderBackend
+  store_local: True
+  store_remote: True
+  store_synchronous: True
+  config:
+    bucket: synapse-storage
+    endpoint_url: https://ceph.berkeley.edu:8443
```
Comment (ja5087) [inline kubernetes/secrets/synapse-config/homeserver.yaml.erb:53]: This does not seem accurate

Comment (expedited) [reply kubernetes/secrets/synapse-config/homeserver.yaml.erb:53]: good catch let me change this rn

Comment (expedited) [reply kubernetes/secrets/synapse-config/homeserver.yaml.erb]: updated

**Comment by nikhiljha** [comment]
oldceph is dead, closing

**Pull request closed without merging**